torch/package/_file_structure_representation.py

```diff
@@ -4,27 +4,47 @@
 from .glob_group import GlobPattern, GlobGroup
 
 
-class Folder:
+class Directory:
+    """A file structure representation. Organized as Directory nodes who have lists of
+    their Directory children."""
+
     def __init__(self, name: str, is_dir: bool):
         self.name = name
         self.is_dir = is_dir
-        self.children: Dict[str, Folder] = {}
+        self.children: Dict[str, Directory] = {}
+
+    def get_dir(self, dirs: List[str]):
+        """Builds path of Directories if not yet built. Return last directory
+        in list.
 
-    def get_folder(self, folders: List[str]):
-        # builds path of folders if not yet built, returns last folder
-        if len(folders) == 0:
+        Args:
+            dirs (List[str]): List of directory names that are treated like a path.
+        """
+        # builds path of directories if not yet built, returns last directory
+        if len(dirs) == 0:
             return self
-        folder_name = folders[0]
-        if folder_name not in self.children:
-            self.children[folder_name] = Folder(folder_name, True)
-        return self.children[folder_name].get_folder(folders[1:])
+        dir_name = dirs[0]
+        if dir_name not in self.children:
+            self.children[dir_name] = Directory(dir_name, True)
+        return self.children[dir_name].get_dir(dirs[1:])
 
-    def add_file(self, file_path):
-        *folders, file = file_path.split("/")
-        folder = self.get_folder(folders)
-        folder.children[file] = Folder(file, False)
+    def add_file(self, file_path: str):
+        """Adds a file to a Directory.
+
+        Args:
+            file_path (str): Path of file to add. Last element is added as a file while
+                other paths items are added as directories.
+        """
+        *dirs, file = file_path.split("/")
+        dir = self.get_dir(dirs)
+        dir.children[file] = Directory(file, False)
 
     def has_file(self, filename: str):
+        """ Returns `True` if a Directory contains the specified file.
```

Comment:
Can you put this in a `Returns` section? That will get it to render properly in the API reference:

https://github.com/pytorch/pytorch/blob/master/torch/package/package_importer.py#L115-L116